solr/solrj/src/java/org/apache/solr/common/cloud/Replica.java

```diff
@@ -437,5 +432,9 @@ public interface ReplicaStateProps {
     String NODE_NAME = "node_name";
     String BASE_URL = "base_url";
     String PROPERTY_PREFIX = "property.";
+    String FORCE_SET_STATE = "force_set_state";
+    Set<String> WELL_KNOWN_PROPS =
+        Set.of(
+            LEADER, STATE, CORE_NAME, CORE_NODE_NAME, TYPE, NODE_NAME, BASE_URL, FORCE_SET_STATE);
```

Comment:
I think `CORE_NODE_NAME` not require. I hope no one setting this. Please make sure in zk.